java/spacewalk-java.spec

```diff
@@ -622,6 +622,21 @@ ln -s -f %{_javadir}/postgresql-jdbc.jar $RPM_BUILD_ROOT$RHN_SEARCH_BUILD_DIR/po
 ln -s -f %{_javadir}/ongres-scram/client.jar $RPM_BUILD_ROOT$RHN_SEARCH_BUILD_DIR/ongres-scram_client.jar
 ln -s -f %{_javadir}/ongres-scram/common.jar $RPM_BUILD_ROOT$RHN_SEARCH_BUILD_DIR/ongres-scram_common.jar
 
+# write an include file for the filelist
+if [ -e %{_javadir}/ongres-stringprep/stringprep.jar ]; then
+    ln -s -f %{_javadir}/ongres-stringprep/stringprep.jar $RPM_BUILD_ROOT$RHN_SEARCH_BUILD_DIR/ongres-stringprep_stringprep.jar
+    ln -s -f %{_javadir}/ongres-stringprep/saslprep.jar $RPM_BUILD_ROOT$RHN_SEARCH_BUILD_DIR/ongres-stringprep_saslprep.jar
+    echo "
+%{serverdir}/tomcat/webapps/rhn/WEB-INF/lib/ongres-stringprep_stringprep.jar
+%{serverdir}/tomcat/webapps/rhn/WEB-INF/lib/ongres-stringprep_saslprep.jar
+%{_prefix}/share/rhn/search/lib/ongres-stringprep_stringprep.jar
+%{_prefix}/share/rhn/search/lib/ongres-stringprep_saslprep.jar
+    " > .mfiles-postgresql
```

Comment:
Just curious. What does `mfiles` mean?

Reply:
I was stealing this name from java packaging. The maybe it is "maven files" or something.